### DISCUSSION TITLE
Having a title

### DISCUSSION THREAD

**Discussion by neldonado** [waschinski/photo-stream#58] [Q&A]
Is there a setting I am missing to create a title bar like I see in this demo?

<img width="1396" alt="image" src="https://user-images.githubusercontent.com/46225185/205468047-2818c6c9-f7e6-4d3d-a155-8223e7c53feb.png">

https://jad.photos/

Also,

How do I change the website description? ![image](https://user-images.githubusercontent.com/46225185/205747172-56ca803b-647a-4b7e-a42d-8dc133e3e659.jpeg)

**Comment by waschinski** [comment]
Pretty sure that's a custom change you see on that website where some HTML has been added manually.

Regarding the description, setting the env variable `DESCRIPTION` should do the job.

**Reply by nickelswitte** [reply]
Would it be possible to implement this into the default repo?

**Reply by waschinski** [reply]
> Would it be possible to implement this into the default repo?

A header (title bar) like shown in the screen? As long as we keep it optional, minimal and slightly customizable, I think it'd be a nice new feature.

**Comment by waschinski** [comment]
Added header enhancement issue #60.
